js/main.js

```diff
@@ -0,0 +1,85 @@
+;(function() {
+
+  /**
```

Comment:
In here are the basic building blocks for filtering the collection of events based on event type, date range, and zip code, as well as plotting it on a map and zooming the map accordingly.